docs/upgrade_skaffold.md

```diff
@@ -64,25 +50,17 @@ deploy:
       - ./deployments/snack.yaml
 ```
 
-In Tilt we similarly associate your image and your Kubernetes YAML through the concept of a service:
+In Tilt we associate your image and your Kubernetes YAML by image tag.
 
 ```python
-  yaml = read_file('./deployments/snack.yaml')
-  service = k8s_service(img, yaml=yaml)
+k8s_resource('snack', 'deployments/snack.yaml')
 ```
 
-5. Return your service!
-
-```python
-  return service
-```
+In Skaffold services are implicitly named from their Kubernetes config. In Tilt services have names given by the first argument to the `k8s_resource` function.
```

Comment:
I'd say "Tilt breaks out resources by image."

I think skaffold does it be pod name, but now I realize I'm not sure...